terraform-local-docker/main.tf

```diff
@@ -0,0 +1,68 @@
+terraform {
+  required_providers {
+    docker = {
+      source  = "kreuzwerker/docker"
+      version = "~> 2.23.1"
+    }
+  }
+}
+
+provider "docker" {}
+
+resource "docker_volume" "db_data" {}
+resource "docker_volume" "wp_data" {}
+
+resource "docker_network" "wp_net_yabs" {
+  name = "wp_net_yabs"
+}
+
+resource "docker_image" "mariadb_image" {
+  name = "mariadb:10.10.2"
+}
+resource "docker_image" "wordpress_image" {
+  name = "wordpress:latest"
+}
+
+resource "docker_container" "mariadb_container" {
+  name  = "db_yabs"
+  image = docker_image.mariadb_image.image_id
+  command      = ["--default-authentication-plugin=mysql_native_password"]
+  restart      = "always"
+  network_mode = "wp_net_yabs"
+  env = [
+    "MYSQL_ROOT_PASSWORD=somewordpress",
+    "MYSQL_DATABASE=wordpress",
+    "MYSQL_USER=wordpress",
+    "MYSQL_PASSWORD=wordpress"
```

Comment:
This are secrets that should not be in github.
Ok for now but it would be better to use variables and set them from outside
[variables terraform](https://developer.hashicorp.com/terraform/language/values/variables)